_posts/en/newsletters/2020-07-22-newsletter.md

```diff
@@ -59,7 +59,30 @@ changes to popular Bitcoin infrastructure software.
 *In this monthly feature, we highlight interesting updates to Bitcoin
 wallets and services.*
 
-FIXME:bitschmidty
+- **Electrum adds Lightning Network and PSBT support:**
+  [Electrum's 4.0.1][electrum release notes] LN support includes local and
+  remote [watchtower][topic watchtowers] capabilities
+  as well as submarine swaps using Electrum Technologies' central server.
+  Electrum has also replaced their partial transaction format with PSBT.
+
+- **Lily Wallet initial release:**
+  [Lily Wallet][lily wallet site] has announced the initial v0.0.1-beta release
+  of their multi-account desktop multisig vault application. Lily Wallet creates
+  2-of-3 bech32 addresses and uses [HWI][topic hwi], HD wallets, and PSBT to transact.
+
+- **Bitcoin Wallet Tracker announced:**
```

Comment:
Note, we previously covered BWT in newsletter 100: https://bitcoinops.org/en/newsletters/2020/06/03/#new-node-to-wallet-middleware

Reply:
I forgot we already covered it. Removed from client/services section. Thanks @harding